js/utils.js

```diff
@@ -415,6 +415,162 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
     }
   }
 
+  function getDateModifiedValues(options) {
+    var isEntryContainsTime = /^([0-1][0-9]|[2][0-3]):([0-5][0-9])$/.test(options.date);
+    var deviceTimeZone = moment.tz.guess();
+    var getDate = options.isDeviceTimeZoneEnabled ? moment.tz.setDefault(deviceTimeZone) : moment.utc;
+
+    switch (options.filterModifier) {
+      case 'today':
+        return {
+          today: moment().startOf('day'),
+          inputDate: isEntryContainsTime
+            ? moment(options.date, 'HH.mm').startOf('day')
```

Comment:
Why have you chosen to use `HH.mm` with `.` and not the standard `:` syntax?

```suggestion
            ? moment(options.date, 'HH:mm').startOf('day')
```

Also, based on the requirements highlight below, if the entry only contains time, it's not meant to be valid, therefore fails the filter check. Why set the input date as `moment().startOf('day')`?

![image](https://user-images.githubusercontent.com/290733/108637212-af903980-7481-11eb-8e01-3928676004a5.png)

Reply:
Fixed